src/components/Hero.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import heroImage from "@/assets/heroImage.svg";
+import Image from "next/image";
+
+const Hero: React.FC = () => {
+  return (
+    <div className="hero bg-slate-900 h-auto md:min-h-screen md:h-[390px]">
+      <div className="hero-content flex-col lg:flex-row-reverse">
+        <Image
+          src={heroImage}
+          alt="hero-image"
+          className="w-full h-[390px] max-w-md rounded-lg shadow-2xl object-cover"
+        />
+        <div className="flex-1 pe-16">
+          <h1 className="text-3xl md:text-5xl font-bold">
+            Stay Ahead of GitHub Trends
+          </h1>
+          <p className="py-8 text-sm md:text-base">
+            Discover the top trending repositories with TrendTrack, the ultimate
+            platform for developers, tech enthusiasts, and curious minds.
+            Explore fresh ideas, cutting-edge tools, and innovative projects—all
+            in one user-friendly place. Stay u`pdated daily and see what's
```

Comment:
fix linter and fix spelling
```suggestion
            in one user-friendly place. Stay updated daily and see what&apos;s
```